src/main/java/main/Menu.java

```diff
@@ -98,42 +86,96 @@ public void showMainMenu() {
 		}
 	}
 
-	public void showOrderSubMenu() {
+	public void showOrderServiceMenu() {
 		boolean back = false;
 		while (!back) {
 			System.out.println("[1] Lista zamówień");
-			System.out.println("[2,OrderNumber] Konkretne zamówienie");
-			System.out.println("[3,clientName,clientSurname,clientAddress,orderStatus] Dodaj zamówienie");
-			System.out.println("[4,OrderNumber] Usuń zamówienie");
-			System.out.println("[5] Cofnij");
+			System.out.println("[2,orderNumber] Konkretne zamówienie");
+			System.out.println("[3,firstName,lastName,address,orderStatus,productId1:quantity1,productId2:quantity2,...] Dodaj zamówienie");
+			System.out.println("[4,orderNumber] Usuń zamówienie");
+			System.out.println("[5,orderNumber,newStatus] Zaktualizuj status zamówienia");
+			System.out.println("[6] Cofnij");
 
 			final String choice = scanner.next();
 			final String[] words = choice.split(",");
 			scanner.nextLine();
 
 			switch (Integer.parseInt(words[0])) {
 				case 1 -> System.out.println(orderService.getAllOrders());
-				case 2 -> System.out.println(orderService.findOrder(words[1]));
+				case 2 -> {
+					final int orderNumberIndex = 1;
```

Comment:
What is this variable for?

Reply:
This variable allows to refer to the appropriate element of the words array that contains the order number without having to repeat index 1 in multiple places in the code.